layout/Library/Application Support/YTMusicUltimate.bundle/es.lproj/Localizable.strings

```diff
@@ -1,91 +1,102 @@
+"OLED_DARK_THEME" = "Tema oscuro OLED";
+"OLED_DARK_THEME_DESC" = "Activa el tema oscuro OLED";
+"OLED_DARK_KEYBOARD" = "Teclado oscuro OLED";
+"OLED_DARK_KEYBOARD_DESC" = "Activa el teclado oscuro OLED";
+"LOW_CONTRAST" = "Bajo contraste";
+"LOW_CONTRAST_DESC" = "Menor contraste para texto y botones";
+
+"NAVBAR_SETTINGS" = "Opciones de la barra de navegación";
+"DONT_STICK_HEADERS" = "No fijar encabezados";
+"DONT_STICK_HEADERS_DESC" = "Despega los paneles fijados en la parte superior de la pantalla";
+"HIDE_HISTORY_BUTTON" = "Ocultar botón de historial";
+"HIDE_HISTORY_BUTTON_DESC" = "Oculta el botón de historial en la barra de navegación";
+"HIDE_CAST_BUTTON" = "Ocultar botón de transmisión";
+"HIDE_CAST_BUTTON_DESC" = "Oculta el botón de transmisión en la barra de navegación";
+"HIDE_FILTER_BUTTON" = "Eliminar botón de filtro de ordenamiento";
+"HIDE_FILTER_BUTTON_DESC" = "Elimina el botón de filtro de Biblioteca/Descargas de la barra de navegación";
+
+"TABBAR_SETTINGS" = "Opciones de la barra de pestañas";
+"STARTUP_TAB" = "Pestaña de inicio";
+"HOME" = "Inicio";
+"SAMPLES" = "Muestras";
+"EXPLORE" = "Explorar";
+"LIBRARY" = "Biblioteca";
+
+"TAB_SETTINGS" = "Configuración de pestañas";
+"REMOVE_TABBAR_LABELS" = "Sin etiquetas en la barra de pestañas";
+"HIDE_HOME" = "Ocultar pestaña Inicio";
+"HIDE_SAMPLES" = "Ocultar pestaña Muestras";
+"HIDE_EXPLORE" = "Ocultar pestaña Explorar";
+"HIDE_LIBRARY" = "Ocultar pestaña Biblioteca";
+
+"APPLY" = "Aplicar";
+"APPLY_DESC" = "Cierra la aplicación para aplicar los cambios";
+
+"LINKS" = "Enlaces";
+"TWITTER" = "Sígueme en Twitter: %@";
+"TWITTER_DESC" = "Sígueme para recibir actualizaciones";
+"DISCORD" = "Únete a Discord";
+"DISCORD_DESC" = "Informa sobre errores y solicita funciones";
+"SOURCE_CODE" = "Código fuente";
+"SOURCE_CODE_DESC" = "Echa un vistazo al código";
+
+"CLOSE" = "Cerrar";
+"WARNING" = "Advertencia";
+"LOGIN_INFO" = "Es necesario reiniciar la aplicación después de iniciar sesión correctamente en la cuenta de Google";
+"APPLY_MESSAGE" = "Esta acción cerrará YouTube Music para aplicar los cambios. ¿Estás seguro?";
+"CANCEL" = "Cancelar";
+"YES" = "Sí";
+"REGIONAL_RESTRICTION" = "Parece que YouTube Music está restringido en tu país. Utiliza una VPN para acceder a YouTube Music.";
+
+"DONT_RUSH" = "No hay que precipitarse";
+"DONT_RUSH_DESC" = "Es necesario reproducir la canción para extraer el enlace";
+"SELECT_ACTION" = "Seleccionar acción";
+"DOWNLOAD_AUDIO" = "Descargar audio";
+"DOWNLOAD_COVER" = "Descargar portada";
+"DOWNLOADING" = "Descargando";
+"OOPS" = "Oops\nalgo salió mal";
+"SAVED_TO_PHOTOS" = "Guardar en Fotos";
+"DONE" = "Hecho";
+
+"DOWNLOADS" = "Descargas";
+"EMPTY" = "El contenido que descargue se mostrará aquí";
+"REMOVE_ALL" = "Eliminar todo";
+"SHARE_ALL" = "Compartir todo";
+"PREPARING" = "Preparando...";
+"DELETE_MESSAGE" = "¿Estás seguro de que quieres eliminar %@?";
+"ALL_DOWNLOADS" = "Descargar todos"; //keep it lowercase
```

Comment:
ALL_DOWNLOADS is used only in DELETE_MESSAGE string. it should be lowercase and supplement the translation.
Example: "Are you sure you want to delete all download audios?"
"¿Estás seguro de que quieres eliminar todos los audios descargados?"

Reply:
This:

"DELETE_MESSAGE" = "Are you sure you want to delete %@?";

change it to this one?

"DELETE_MESSAGE" = "Are you sure you want to delete all downloaded audios?"

Reply:
a ok ok, I think I got it, the ALL_DOWNLOADS is a string that in context is "delete all downloaded audios" isn't it?